src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck1 = new Truck();
+        Machine bulldozer1 = new Bulldozer();
+        Machine excavator1 = new Excavator();
+
+        Machine[] machines = new Machine[] {truck1, bulldozer1, excavator1};
```

Comment:
Make a note that you can call new Excavator(), new Bulldozer(), new Truck() explicitly in array initialization without temporary variables.